go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	github.com/multiformats/go-multiaddr-dns v0.3.1
 	github.com/natefinch/lumberjack v2.0.0+incompatible
 	github.com/pborman/uuid v1.2.0
-	github.com/pelletier/go-toml v1.2.0
+	github.com/pelletier/go-toml v1.9.3
```

Comment:
Dependency `go-toml` has to be updated otherwise array default values can't be set in `cmd/harmony/config_migrations.go`

Reply:
Nice! Thanks!